<launch>
  <arg name="use_builtin_visual_odom" default="true"/>

  <include file="$(find mrover)/launch/zed.launch">
    <arg name="use_builtin_visual_odom" value="$(arg use_builtin_visual_odom)"/>
  </include>

  <node pkg="nodelet" type="nodelet" name="perception_nodelet_manager" required="true"
        args="manager" output="screen"/>

  <node name="panorama" pkg="mrover" type="panorama.py" output="screen"/>

  <node pkg="tf" type="static_transform_publisher" name="base_link_publisher"
        args="0 0 0 0 0 0 base_link zed_base_link 100"/>
</launch>
